### DISCUSSION TITLE
Github is not a social media

### DISCUSSION THREAD

**Discussion by mspronesti** [community/community#66184] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I thought the community clarified a while ago that Github is not intended to be a social media. This new feed  doesn't show anything useful and none of it looks related to the people I follow.
Please bring the previous one back.

**Comment by EnzoMartin** [comment]
Agreed. I'd like to pin/favorite repositories and get a feed of the PRs, Commits, and Releases happening to those repositories instead of this "Feed"

**Comment by kurtmckee** [comment]
Agreed. I've had to turn off everything, resulting in a blank default page.

Please bring the previous one back.

**Reply by beeman** [reply]
Indeed. 

Seeing this screenshot is better than the garbage feed they created 'for me' 😤
![image](https://github.com/community/community/assets/36491/95c94d9a-06ce-45aa-beed-da89b73fe0d0)

**Reply by kurtmckee** [reply]
Yep! I've modified my Firefox start page to take me directly to my GitHub notifications.
